### DISCUSSION TITLE
Raspberrymatic VM system update aus HA

### DISCUSSION THREAD

**Discussion by ZTHawk** [OpenCCU/OpenCCU#2540] [🇩🇪 Fragen&Antworten] [answered]
Moin,
Ich nutze für HA das addon https://github.com/danielperna84/hahomematic.
Raspberrymatic habe ich in einer eigenen VM laufen (also nicht als HA addon).
Es scheint kein API zu geben per XML-RPC über ein neues Systemupdate in Kentniss gesetzt zu werden und/oder dieses einzuspielen.

Ist dies korrekt? Wenn ja, gibt es Pläne dies hinzuzufügen?
Gibt es Alternativen?

**Comment by jens-maus** [comment]
Was meinst du denn mit "Systemupdate"? Eine neue RaspberryMatic Version? Diese Update-Information stellt dir doch im Falle des HomeAssistant Add-ons, HomeAssistant selbst zur Verfügung. So wie z.b. heute aktuell nachdem ich das neue Update freigegeben habe. Siehe:

<img width="980" alt="Bildschirmfoto 2023-11-30 um 18 13 00" src="https://github.com/jens-maus/RaspberryMatic/assets/5887769/29e44660-620a-420a-8df0-865798ef4c9c">

... und natürlich immer schön brav vor jedem Update ein Backup machen ;-)

**Reply by ZTHawk** [reply]
Ja das stimmt, aber:
Ich nutze wie oben erwähnt nicht das HA addon, sonder habe Raspberrymatic in einer separaten VM (damit ich auch das HmIP-HAP als ~~AccessPoint~~ Gateway nutzen kann).
Ich würde gerne von dieser separaten Instanz wissen, ob für sie ein Update vorhanden ist.

Für einzelne HmIP Geräte funktioniert es. Die "CCU" meldet aber nicht, dass ein Systemupdate vorhanden ist (am liebsten wie oben auf dem Screenshot).
Zumindest wurde dies hier sagt, dass es nicht geht: https://github.com/danielperna84/hahomematic/discussions/1306.

**Reply by jens-maus** [reply, marked as answer]
Und genau so ist das: Geht nicht und ist auch nicht vorgesehen. Wenn du über RaspberryMatic/CCU Updates informiert werden willst musst du das aus RaspberryMatic heraus mit entsprechende Mitteln tun (in verschiedenen Foren zu finden) Das wie vorgeschlagen via XMLRPC auszuliefern oder sonst wie ist der falsche Weg.
